src/app/component/header/header.component.html

```diff
@@ -1,10 +1,10 @@
 <nav class="navbar fixed-top navbar-expand-sm navbar-light bg-light" style="padding: 0px;">
   <div class="container-fluid">
     <a href="#" class="navbar-brand h1" routerLink="/dashboard" style="margin: 0px;">
-      <img class="d-inline-block" src="assets/img/logo1.png" width="35" height="35" />
+      <img class="d-inline-block" src="assets/img/logo1.png" alt="Jyoti Techn" width="35" height="35" />
```

Comment:
Could you please move this style width and height to external css

Reply:
I have already Done  inline CSS to external CSS

Reply:
i cant see your changes  could you pleaes check this again as it is still showing